machine_learning_hep/utils/hist.py

```diff
@@ -248,3 +251,9 @@ def sum_hists(hists, name = None):
 def ensure_sumw2(hist):
     if hist.GetSumw2N() < 1:
         hist.Sumw2()
+
+
+def print_histogram(hist):
+    n_bins = hist.GetNbinsX()
+    print(f"{hist.GetName()}, {n_bins} bins, {hist.GetXaxis().GetXmin()} to {hist.GetXaxis().GetXmax()}")
+    print([hist.GetBinContent(i + 1) for i in range(n_bins)])
```

Comment:
I would prefer to keep only genuine ROOT workarounds/extensions in this file, not printing/plotting stuff. And I don't see why this is needed in the first place, you can just use `h.Print("all")` or whatever option gives you the verbosity you need.